actors/player_kinematic_character.gd

```diff
@@ -3,4 +3,4 @@ extends "kinematic_character.gd"
 func handle_input():
 	pass
 func _input(event):
-	state_machine.get_state().handle_input(self, event)
\ No newline at end of file
+	state_machine.state.handle_input(self, event)
```

Comment:
move this to a state_change `signal` so that people that grab this script can use it without the need of a state_machine